components/brave_wallet/browser/brave_wallet_utils_unittest.cc

```diff
@@ -761,20 +761,6 @@ TEST(BraveWalletUtilsUnitTest, GetNetworkURLTest) {
             GetNetworkURL(&prefs, chain2.chain_id, mojom::CoinType::ETH));
 }
 
-TEST(BraveWalletUtilsUnitTest, GetInfuraSubdomainForKnownChainId) {
-  TestingPrefServiceSimple prefs;
-  prefs.registry()->RegisterDictionaryPref(kBraveWalletCustomNetworks);
-  prefs.registry()->RegisterBooleanPref(kSupportEip1559OnLocalhostChain, false);
-
-  std::vector<mojom::NetworkInfoPtr> known_chains;
-  GetAllKnownEthChains(&prefs, &known_chains);
-  for (const auto& chain : known_chains) {
-    auto subdomain = GetInfuraSubdomainForKnownChainId(chain->chain_id);
```

Comment:
Don't understand why there is requirement for any known network to have infura subdomain. By now it looks like not a product requirement but just behavior fixed by tests. 
If this test should be kept alive please someone explain what it should test ;)